### DISCUSSION TITLE
Error: from spacy.gold import GoldParse No name GoldParse in Module spacy.gold

### DISCUSSION THREAD

**Discussion by NightFallLegacy** [explosion/spaCy#7369] [Help: Installation] [answered]
Hi,

When I try to train a blank model and use GoldParse to store training annotations I get the following error. `No name 'GoldParse' in module 'spacy.gold'pylint(no-name-in-module)`. I used the `dir(spacy.gold)` command to check if GoldParse is there and it is. However, I still get this error. When I run the training code it trains and returns the loss values to me however it abruptly ends at different stages of training. I assume this is due to the GoldParse portion of my code because I have an `nlp.to_disk` command at the end and the model is never saved at the specified path.  I would appreciate if someone could help me out with this :) I have looked into this issue for a while but to no avail.

**Comment by svlandeg** [comment]
What versions of `spacy` and `spacy-transformers` are you using, and how did you install them?

**Comment by NightFallLegacy** [comment]
I've gotten this error in two different versions of spacy. I'm not using the spacy-transformers in this build.

Spacy: 2.3.0(current version), 2.0.16
Python: 3.8.4

I installed all my packages using the pip installer

**Comment by svlandeg** [comment]
Could there be conflicting versions of spaCy installed in the same environment? Have you tried starting from a fresh, clean Python environment?

**Comment by NightFallLegacy** [comment]
I will test the code in a google colab and get back to you on that

**Comment by NightFallLegacy** [comment, marked as answer]
I set up the google colab environment and tested it, it made it past the first iteration unlike all my previous attempts so I'm optimistic that it will finish training.  I think the issue arose from a conflicting environment as you mentioned. Thanks for your help :)

**Comment by svlandeg** [comment]
Great, happy to hear you got it sorted out!
